tests/e2e/test_e2e.py

```diff
@@ -275,13 +276,19 @@ def regression_bounded(model, test_fraction=0.02):
         regression(linear_model.OrthogonalMatchingPursuitCV()),
         regression(linear_model.PassiveAggressiveRegressor(
             random_state=RANDOM_SEED)),
+        regression(linear_model.PoissonRegressor()),
         regression(linear_model.RANSACRegressor(
             base_estimator=tree.ExtraTreeRegressor(**TREE_PARAMS),
             random_state=RANDOM_SEED)),
         regression(linear_model.Ridge(random_state=RANDOM_SEED)),
         regression(linear_model.RidgeCV()),
         regression(linear_model.SGDRegressor(random_state=RANDOM_SEED)),
         regression(linear_model.TheilSenRegressor(random_state=RANDOM_SEED)),
+        regression(linear_model.TweedieRegressor(power=0.0)),
+        regression(linear_model.TweedieRegressor(power=1.0)),
+        regression(linear_model.TweedieRegressor(power=1.5)),
+        regression(linear_model.TweedieRegressor(power=2.0)),
+        regression(linear_model.TweedieRegressor(power=3.0)),
```

Comment:
Refer to https://github.com/scikit-learn/scikit-learn/blob/483cd3eaa3c636a57ebb0dc4765531183b274df0/sklearn/linear_model/_glm/glm.py#L525-L543.